dashboard/app/models/census/state_cs_offering.rb

```diff
@@ -83,6 +83,14 @@ class Census::StateCsOffering < ApplicationRecord
   STATES_USING_FORMAT_V2_IN_2017_18 = %w(
     CA
     HI
+    IL
```

Comment:
You don't have to remove these from the 2018_19 list because we have new files for 2018-19 for each of these states, yeah? Is there some specific order of operations you have to follow in order not to have things break?

Reply:
Looks like these changes won't cause any seeding issues like what Bethany alerted us to earlier today.

Reply:
Unfortunately it will because of this routine
https://github.com/code-dot-org/code-dot-org/blob/fa40db76466a3cd34e02f3f5a72a635f63c3ebf6/dashboard/app/models/census/state_cs_offering.rb#L1552-L1563

Reply:
Let's discuss it next week.

Reply:
🤦 I looked at those lines at was like "no problem"! Obviously will cause a problem. Next week sounds good!